src/main/java/com/searchservice/app/domain/utils/UploadDocumentUtil.java

```diff
@@ -19,53 +19,72 @@
 @Data
 @Component
 public class UploadDocumentUtil {
-
-	private static final String APPLICATION_JSON = "application/json";
-
+	  private static String contentType="application/json";
 	private final Logger log = LoggerFactory.getLogger(UploadDocumentUtil.class);
 
 	private String baseSearchUrl;
 	private String tableName;
 	private String content;// "[{'name': 'karthik1'},{'name': 'karthik2'}]"
-
+  
 	public UploadDocumentSearchUtilRespnse commit() {
+
 		OkHttpClient client = new OkHttpClient();
-		MediaType mediaType = MediaType.parse(APPLICATION_JSON);
+		MediaType mediaType = MediaType.parse(contentType);
 		RequestBody body = RequestBody.create(mediaType, content);
 
 		String url = baseSearchUrl + "/" + tableName + "/update?";
 		url += "commit=true";
 
-		return processUploadDocumentRequest(client, body, url);
+		Request request = new Request.Builder().url(url).method("POST", body)
+				.addHeader("Content-Type", contentType).build();
+
+		try {
+
+			Response response = client.newCall(request).execute();
+			if (response.code() != 400) {
+				return new UploadDocumentSearchUtilRespnse(true, "Document Added Successfully!");
+			} else {
+				throw new CustomExceptionHandler(400,HttpStatusCode.BAD_REQUEST_EXCEPTION, "Document not uploaded!");
+			}
+		} catch (IOException e) {
+			log.error(e.toString());
+
+			return new UploadDocumentSearchUtilRespnse(false, "Document not uploaded! IOException.");
+
+		}
+
 	}
 
 	public UploadDocumentSearchUtilRespnse softcommit() {
+
 		OkHttpClient client = new OkHttpClient();
-		MediaType mediaType = MediaType.parse(APPLICATION_JSON);
+
+		MediaType mediaType = MediaType.parse(contentType);
+
 		RequestBody body = RequestBody.create(mediaType, content);
 
 		String url = baseSearchUrl + "/" + tableName + "/update?";
+
 		url += "softCommit=true";
 
 		log.debug("SOFT COMMIT");
 
-		return processUploadDocumentRequest(client, body, url);
-	}
-	
-	private UploadDocumentSearchUtilRespnse processUploadDocumentRequest(OkHttpClient client, RequestBody body, String url) {
```

Comment:
WHY this method is removed? Any reason?

Reply:
I didnot remove that.

Reply:
It's there in main-dev branch, right? Pls check that properly